sickbeard/server/web/config/search.py

```diff
@@ -69,7 +69,7 @@ def saveSearch(self, use_nzbs=None, use_torrents=None, nzb_dir=None, sab_usernam
         sickbeard.BACKLOG_DAYS = try_int(backlog_days, 7)
 
         sickbeard.CACHE_TRIMMING = config.checkbox_to_value(cache_trimming)
-        sickbeard.MAX_CACHE_AGE = float(max_cache_age)
+        sickbeard.MAX_CACHE_AGE = try_int(max_cache_age)
```

Comment:
0 since in cache trim i do `if days:` so that it would skip the trim instead of trimming all but 1 day.